fearless/Modules/Staking/RecommendedValidators/RecommendedValidatorsProtocols.swift

```diff
@@ -23,13 +23,21 @@ protocol RecommendedValidatorsInteractorOutputProtocol: class {
 }
 
 protocol RecommendedValidatorsWireframeProtocol: AlertPresentable, ErrorPresentable {
-    func proceed(from view: RecommendedValidatorsViewProtocol?, result: PreparedNomination)
+    func proceed(from view: RecommendedValidatorsViewProtocol?,
+                 targets: [SelectedValidatorInfo],
+                 maxTargets: Int)
+
     func showRecommended(from view: RecommendedValidatorsViewProtocol?,
                          validators: [ElectedValidatorInfo],
                          maxTargets: Int)
+
     func showCustom(from view: RecommendedValidatorsViewProtocol?, validators: [ElectedValidatorInfo])
 }
 
 protocol RecommendedValidatorsViewFactoryProtocol: class {
```

Comment:
definition `RecommendedValidatorsViewFactoryProtocol` is never used, is it necessary to define such "..ViewFactoryProtocol" among project?

Reply:
Yep, the protocol is not used because we don't pass factory around and just consume it's static methods. I think we can just get rid of this but we need to fix template generator.